gateways/orbital/request_builders.go

```diff
@@ -0,0 +1,84 @@
+package orbital
+
+import (
+	"encoding/xml"
+	"strconv"
+
+	"github.com/BoltApp/sleet"
+)
+
+func buildAuthRequest(authRequest *sleet.AuthorizationRequest) Request {
+
+	amount := authRequest.Amount.Amount
+	exp := strconv.Itoa(authRequest.CreditCard.ExpirationYear) + strconv.Itoa(authRequest.CreditCard.ExpirationMonth)
+	code := currencyMap[authRequest.Amount.Currency]
+
+	body := RequestBody{
+		IndustryType:     IndustryTypeEcomm,
+		MessageType:      MessageTypeAuth,
+		BIN:              BINStratus,
+		TerminalID:       TerminalIDStratus,
+		AccountNum:       authRequest.CreditCard.Number,
+		Exp:              exp,
+		CurrencyCode:     code,
+		CurrencyExponent: CurrencyExponentDefault,
+		CardSecValInd:    CardSecPresent,
```

Comment:
From the NewOrder message definition it seems like this value should be null if the transaction is not a Visa or Discover transaction. I'm not sure if we have a way to do this currently, so I think this is fine for now but I just want to make note of it